hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsScheduler.java

```diff
@@ -127,6 +126,10 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
       status.getMessage(),
       liveState.getStagingTasksSize()));
 
```

Comment:
I've added an if block to only notify the Reconciler of StatusUpdates when it is is reconciling.  In the future this can be modified if LiveState and PersistentState etc. become Observers as well.